data/src/main/java/com/globant/equattrocchio/data/ResultToImageListMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.globant.equattrocchio.data;
+
+import android.support.annotation.NonNull;
+
+import com.globant.equattrocchio.data.response.Image;
+import com.globant.equattrocchio.data.response.Result;
+import com.globant.equattrocchio.domain.models.ImageList;
+
+import java.util.ArrayList;
+
+public class ResultToImageListMapper {
+    @NonNull
+    public static ImageList map(@NonNull Result result) {
+        ImageList imageList = new ImageList();
+        imageList.setImages(new ArrayList<com.globant.equattrocchio.domain.models.Image>());
+
+        if (result.getImages() != null) {
+            for (Image img : result.getImages()) {
```

Comment:
Perhaps rename the Image from Responses to don't need to use qualified names.

Reply:
Done